code/datums/tutorial/marine/medical_basic.dm

```diff
@@ -0,0 +1,170 @@
+	message_to_player("Good. Now, when you normally take damage, you will also feel <b>pain</b>. Pain slows you down and can knock you out if left unchecked.")
+	update_objective("")
+	var/mob/living/living_mob = tutorial_mob
+	living_mob.pain.apply_pain(PAIN_CHESTBURST_STRONG)
+	addtimer(CALLBACK(src, PROC_REF(pain_tutorial)), 4 SECONDS)
+
+/datum/tutorial/marine/medical_basic/proc/pain_tutorial()
+	message_to_player("<b>Tramadol</b> is used to reduce your pain. Inject yourself with the <b>tramadol EZ autoinjector</b>.")
+	update_objective("Inject yourself with the tramadol injector.")
+	var/obj/item/reagent_container/hypospray/autoinjector/tramadol/skillless/one_use/pain_injector = new(loc_from_corner(0, 4))
+	add_to_tracking_atoms(pain_injector)
+	add_highlight(pain_injector)
+	RegisterSignal(tutorial_mob, COMSIG_LIVING_HYPOSPRAY_INJECTED, PROC_REF(on_pain_inject))
+
+/datum/tutorial/marine/medical_basic/proc/on_pain_inject(datum/source, obj/item/reagent_container/hypospray/injector)
+	SIGNAL_HANDLER
+
+	if(!istype(injector, /obj/item/reagent_container/hypospray/autoinjector/tramadol/skillless/one_use))
+		return
+
+	UnregisterSignal(tutorial_mob, COMSIG_LIVING_HYPOSPRAY_INJECTED)
+	TUTORIAL_ATOM_FROM_TRACKING(/obj/item/reagent_container/hypospray/autoinjector/tramadol/skillless/one_use, pain_injector)
+	remove_highlight(pain_injector)
+	message_to_player("Good. Keep in mind that you can overdose on chemicals, so don't inject yourself with the same chemical too much too often. In the field, injectors have 3 uses.")
+	update_objective("Don't overdose! Generally, 3 injections of a chemical will overdose you.")
+	var/mob/living/living_mob = tutorial_mob
+	living_mob.pain.apply_pain(-PAIN_CHESTBURST_STRONG) // just to make sure
+	addtimer(CALLBACK(src, PROC_REF(bleed_tutorial)), 4 SECONDS)
+
+/datum/tutorial/marine/medical_basic/proc/bleed_tutorial()
+	message_to_player("You can sometimes start <b>bleeding</b> from things like bullets or slashes. Losing blood will accumulate <b>oxygen</b> damage, eventually causing death.")
+	update_objective("")
+	var/mob/living/carbon/human/human_mob = tutorial_mob
+	var/obj/limb/chest/mob_chest = locate(/obj/limb/chest) in human_mob.limbs
+	mob_chest.add_bleeding(damage_amount = 15)
+	addtimer(CALLBACK(src, PROC_REF(bleed_tutorial_2)), 4 SECONDS)
+
+/datum/tutorial/marine/medical_basic/proc/bleed_tutorial_2()
+	message_to_player("Bleeding wounds can clot themselves over time, or you can fix it quickly with <b>gauze</b>. Pick up the gauze and click on yourself while targeting your <b>chest</b>.")
+	update_objective("Gauze your chest, or let it clot on its own.")
+	var/obj/item/stack/medical/bruise_pack/two/bandage = new(loc_from_corner(0, 4))
+	add_to_tracking_atoms(bandage)
+	add_highlight(bandage)
+	var/mob/living/carbon/human/human_mob = tutorial_mob
+	var/obj/limb/chest/mob_chest = locate(/obj/limb/chest) in human_mob.limbs
+	RegisterSignal(mob_chest, COMSIG_LIMB_STOP_BLEEDING, PROC_REF(on_chest_bleed_stop))
```

Comment:
That didn't seem to work properly when i tested, chest healed on its own then i was stuck at this tutorial stage

Reply:
fixed